### PR TITLE
Kh/goodness of fit metrics

### PR DESCRIPTION
**Pull request by khenisey**

### CODE REVIEW

File: model/model.wl [hunk 1]
```diff
@@ -569,9 +570,12 @@ evaluateState[state_]:= Module[{distance,sol,params,longData,thisStateData,model
 	  {2,#["day"],(#["positiveIncrease"]/params["population"])//N}&/@thisStateData
 	],#[[3]]>0&];
 	
-	(* assume a constant relative variance for death and PCR rates and weight by 1/sigma^2. *)
+	(* set weights for each datapoint in longData *)
+	(* apply a week over week weight reduction, i.e. 0.75 indicates that a data point at day 0 is weighted 75% as strongly as a data point at day 7. *)
+	(* assume each datapoint otherwise has a constant relative variance (Poissan) for both death and PCR rates. *)
 	(* the constant factor of the population shouldn't matter, but the fit chokes if the weights are too small *)
-	dataWeights=((params["population"]#[[3]])^-1)&/@longData;
+	weekOverWeekWeight = .75
```
Comment (wbunting) [inline model/model.wl:577]: nice!

File: model/model.wl [hunk 2]
```diff
@@ -590,10 +594,12 @@ evaluateState[state_]:= Module[{distance,sol,params,longData,thisStateData,model
 	(* if we cannot get smooth enough then use Nelder-Mead Post-processing \[Rule] false *)
 	
 	fitParams=Exp[#]&/@KeyMap[ToString[#]&, Association[fit["BestFitParameters"]]];
+	(* TODO: try using VarianceEstimatorFunction\[Rule](1)& *)
 	standardErrors=Exp[#]&/@KeyMap[ToString[#]&, AssociationThread[{r0natural,importtime},
 	     fit["ParameterErrors",
 	     ConfidenceLevel->0.97]]];
-	     
+	gofMetrics=goodnessOfFitMetrics[fit["Residuals"],longData];
```
Comment (wbunting) [inline model/model.wl:601]: can we compute this above and then dump to CSV before we return? 

Something like `Export["model/test/fit.csv", goodnesOfFitMetrics[...]]`

Comment (khenisey) [reply model/model.wl:601]: The gof metrics are per state, so if we want to have a CSV (and ultimately plots) of all these things, we'll either have to refactor a bit to separate out the simulation section or duplicate everything that goes before the simulation and run that separately.

Does that make sense?

Comment (wbunting) [reply model/model.wl:601]: My PR refactors the export calls per state!

**Pull request merged**